Development/src/pages/devices/DevicesShow.js

```diff
@@ -32,12 +40,72 @@ export const DevicesShow = props => {
 };
 
 const DevicesShowView = props => {
+    const { record } = useRecordContext();
+    const [useIS08API, setUseIS08API] = useState(false);
+    useEffect(() => {
+        if (get(record, '$channelMappingAPI') !== undefined) {
+            setUseIS08API(true);
+        } else {
+            setUseIS08API(false);
+        }
+    }, [record]);
+
+    const theme = useTheme();
+    const tabBackgroundColor =
+        theme.palette.type === 'light'
+            ? theme.palette.grey[100]
+            : theme.palette.grey[900];
+    return (
+        <>
+            <div style={{ display: 'flex' }}>
+                <Paper
+                    style={{
+                        alignSelf: 'flex-end',
+                        background: tabBackgroundColor,
+                    }}
+                >
+                    <Tabs
+                        value={props.location.pathname}
+                        indicatorColor="primary"
+                        textColor="primary"
+                    >
+                        <Tab
+                            label="Summary"
+                            value={`${props.match.url}`}
+                            component={Link}
+                            to={`${props.basePath}/${props.id}/show/`}
+                        />
+                        {['Active_Matrix', 'Staged_Matrix'].map(label => (
```

Comment:
Although we generally try to stick to the NMOS terminology rather than user friendly terms, in this case I agree it seems completely correct to use *Matrix* rather than *Map*, especially as you are rendering the map as a matrix! 🎉😁

Reply:
In the call we decided to make the URL keys be `active_map` and `staged_map` and the UI show 'Active Map' and 'Staged Map'.